src/arithmetic.jl

```diff
@@ -40,17 +32,11 @@ end
 
 ## zero and one ##
 for T in (:Taylor1, :TaylorN), f in (:zero, :one)
-    @eval begin
-        ($f)(a::$T) = $T([($f)(a[1])], a.order)
-
-        ($f)(a::$T, order) = $T([($f)(a[1])], order)
-    end
+    @eval ($f)(a::$T) = $T(($f)(a[1]), a.order)
```

Comment:
(In the future) we should think if `TaylorN` can literally be written in terms of `Taylor1` to avoid all this duplication of similar methods.